### PR TITLE
feat: #607 - price on the incorporation item details

### PR DESCRIPTION
**Pull request by andregoncalves**
screen shot 2019-02-11 at 17 30 28

Price on the incorporation item details (invision 701)

### CODE REVIEW

**Comment by andregoncalves** [comment]
@rodrigopavezi don't review it jut yet, I just noticed that the KEY value is not properly formatted.

File: src/renderer/marketplace/incorporation/detail-view/index.jsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import injectSheet from 'react-jss';
```
Comment (rodrigopavezi) [inline src/renderer/marketplace/incorporation/detail-view/index.jsx]: @andregoncalves is there a need to react-jss here?

Comment (andregoncalves) [reply src/renderer/marketplace/incorporation/detail-view/index.jsx]: Nice catch @rodrigopavezi , some old incorporations code still used react-jss.
I ended up refactoring and making the same change in other components as well.

Comment (rodrigopavezi) [reply src/renderer/marketplace/incorporation/detail-view/index.jsx]: @andregoncalves awesome. cheers

**Pull request merged**